src/components/post.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+const Post = ({ children }) => {
```

Comment:
Consider change component name to `Posts` and use the same logic like `Tiles` component. Also add `.yml` file and fill there all posts data. To compare take a look in `courses.yml`.